### ISSUE TITLE
Does anyone have an address for the deployment of the smart contract on Goerli or Sepolia?

### ISSUE THREAD

**Issue by Aristo21** [alchemyplatform/nft-minter-tutorial#17]
I guess current contract address points to the original contract on Ropsten. When I run the project, it mints successfully but the etherscan doesn't seem to recognise an NFT creation (no Token ID f.ex.). Does anyone have an address for the deployment of the smart contract on Goerli or Sepolia?

**Comment by Ahmedouu** [comment]
yeah there is something weird here I guess, I tried to find the contract to redeploy it but i couldn't ,.. and i can find the transaction if metamask is connected to the same provider as the app by clicking on view on block explorer inside metamask but if metamask is not on the same provider it still mints which is odd but no transaction on block explorer ...
